release_notes/3057-release-notes.adoc

```diff
@@ -0,0 +1,101 @@
+[id="3057-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes version 3.0.57
+include::modules/common-attributes.adoc[]
+:context: 3057-release-notes
+
+toc::[]
+
+[role="_abstract"]
+{product-title} version 3.0.57 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* March 18, 2021
+
+== New features
+
+=== Scan inactive images
+
+{product-title} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
+You can now add inactive (undeployed) images for automatic scanning.
+//For more details, see link:/docs/manage-vulnerabilities/#scan-inactive-images[Scan inactive images].
+
+== Important bug fixes
+
+* *ROX-6085*: Previously, setting Central log level by using the `roxctl` CLI's `debug log` command would not work sometimes.
+You could also set unacceptable values for log level (for example, `Trace`), which did not affect the log level.
+
+* *ROX-6302*: The *Violations* view where loading too many violations would sometimes crash the RHACS portal page.
```

Comment:
Just checking, is "where" the right word? I'm having trouble understanding what the sentence is trying to say. Should it be "Loading too many violations on the **Violations** view would sometimes..."